meinberlin/apps/kiezradar/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
```

Comment:
this file should be removed :)

Reply:
yes, was generated when i created the app